pkg/services/ngalert/api/api_provisioning_test.go

```diff
@@ -242,11 +243,13 @@ func TestProvisioningApi(t *testing.T) {
 			t.Run("PUT returns 400", func(t *testing.T) {
 				sut := createProvisioningSrvSut(t)
 				rc := createTestRequestCtx()
-				insertRule(t, sut, createTestAlertRule("rule", 1))
-				rule := createInvalidAlertRule()
-
-				response := sut.RoutePutAlertRule(&rc, rule, "rule")
+				uid := "123123"
+				rule := createTestAlertRule("rule", 1)
+				rule.UID = uid
+				insertRule(t, sut, rule)
 
+				rule = createInvalidAlertRule()
+				response := sut.RoutePutAlertRule(&rc, rule, uid)
```

Comment:
This is such a tiny nitpick it's unreal, but the original test had careful newlines to separate setup/act/assert stages and make the test a little more readable as to "what" it's testing. Could we readjust the spacing:
```
insertRule()
rule = createInvalidAlertRule()

response := sut.RoutePutAlertRule(...

require.Equal(...

Reply:
Done